web/styles/_guidance-conversion.scss

```diff
@@ -0,0 +1,46 @@
+@use '@nice-digital/nds-core/scss/spacing';
+@use '@nice-digital/nds-core/scss/utils';
+
+/* stylelint-disable selector-no-qualifying-type, selector-class-pattern */
+// shouldn't be using tags before classes in selectors
+// shouldn't be using class names with hyphens
+// NB. Neither are generated in next-web
+
+.chapter {
+  div.itemizedlist.indented,
+  p.numbered-paragraph {
+    margin-left: utils.rem(spacing.$xx-large);
+  }
+
+  div.itemizedlist.indented .paragraph-number,
+  p.numbered-paragraph .paragraph-number {
+    float: left;
+    margin-left: utils.rem(-(spacing.$xx-large));
+  }
+
+  article.numbered-paragraph {
+    display: flex;
+
+    > {
+      h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6 {
+        flex: 0 0 utils.rem(spacing.$xx-large);
+        font-family: inherit;
+        font-size: inherit;
+        font-weight: normal;
+        line-height: 1.6;
+        margin-top: 0;
+        padding-right: 10px;
+      }
+    }
+
+    p:first-child {
+      margin-top: 0;
+    }
+  }
+}
+/* stylelint-enable selector-no-qualifying-type, selector-class-pattern */
```

Comment:
Really minor thing - don't think you need to re-enable linting rules after they've been disabled, as the disable statements are only scoped to the file that they're added to. Won't cause any issues being here, but might as well remove it!

Reply:
No worries; was following the docs on here:

https://github.com/stylelint/stylelint/blob/main/docs/user-guide/ignore-code.md

Wasn't sure what would happen subsequently without the enable line at the end given that it's a partial. Will remove now.